src/test/java/test/reports/EmailableReporterTest.java

```diff
@@ -19,38 +19,38 @@
 public class EmailableReporterTest extends SimpleBaseTest {
     private SecurityManager manager;
 
-    @BeforeClass (alwaysRun = true)
+    @BeforeClass (alwaysRun=true)
```

Comment:
Sorry I should have been more specific. I mean we write it `@BeforeClass(alwaysRun = true)`.